Sources/ContentView.swift

```diff
@@ -5,25 +5,32 @@ struct ContentView: View {
     @State
     var repositories = [Repository]()
     @State
-    private var searchTerm: String = "php"
+    private var searchTerm: String = ""
+    @State
+    private var isError = false
+    @State
+    private var message = ""
```

Comment:
Same here. Message could mean anything. Rename it to contain more detailed description.
errorMessage would be perfect in this particular case.